.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: File a bug report
+labels: ["Type: bug"]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug, including details regarding any error messages, version, and platform.
+      description: Please include what you expected.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component
+      description: Which component is principally affected?
+      options:
+        - Archery
+        - Benchmarking
+        - C
+        - C#
+        - C++
+        - C++ - Gandiva
+        - C++ - Plasma
+        - Compute IR
```

Comment:
This one should be removed.

Reply:
Removed as part of https://github.com/apache/arrow/pull/14688.